vue2-vuetify/src/layouts/ArrayLayoutRenderer.vue

```diff
@@ -40,7 +40,13 @@
     <v-card-text>
       <v-container justify-space-around align-content-center>
         <v-row justify="center">
-          <v-expansion-panels accordion focusable v-model="currentlyExpanded">
+          <v-expansion-panels
+            accordion
+            :focusable="focusable"
+            v-bind="vuetifyProps('v-expansion-panels')"
+            v-model="currentlyExpanded"
+            flat
```

Comment:
We should handle the `flat` in the same way as `focusable` but with a default of `false`.

Reply:
Agreed, and it provides a pattern for setting other default values for props people may wish to override via the `config.vuetify` option.